README.md

```diff
@@ -103,104 +81,34 @@ Commands:
 Planners have their own options and defaults, sharing most of them in common:
 
 ```bash
-$ ./nix-installer install linux-multi --help
+$ curl -L https://install.determinate.systems/nix | sh -s -- install linux-multi --help
 A standard Linux multi-user install
 
 Usage: nix-installer install linux-multi [OPTIONS]
 
 Options:
-      --channel [<CHANNELS>...]
```

Comment:
IMHO we should snip all of these options if we're snipping any.

Reply:
I left the two that we configure in the next code sample, so as to demonstrate how they look and how to set them in two different ways.